server/app.py

```diff
@@ -840,7 +859,7 @@ def payout_paid(event):
         # get the Opportunity Ids that go with those charges
         sfc = SalesforceConnection()
         opps_to_update = sfc.query(query)
-        opps_to_update = [o["Id"] for o in opps_to_update]
+        opps_to_updkate = [o["Id"] for o in opps_to_update]
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling. -->
**typo?** `opps_to_updkate`